lib/OpenQA/Log.pm

```diff
@@ -174,6 +174,11 @@ sub add_log_channel {
 # The default format for logging
 sub log_format_callback {
     my ($time, $level, @lines) = @_;
+    
+    foreach ( @lines ) {
+        $_ =~ s/(?<!\A)^ */  /gm;
+    }
```

Comment:
please apply the same as in os-autoinst for the code style

Reply:
Done